content/blog/nuttx-getting-started/index.md

```diff
@@ -193,9 +193,10 @@ git clone https://github.com/apache/nuttx-apps apps
 
 ### Building an App to Connect to a Wi-Fi network
 
-NuttX provides ready-to-use board default configurations that enable the required config (from Kconfig) for a use scenario, such as Wi-Fi or I2C. You can list all available configurations for the ESP32 DevKitC V4 board using the following command, inside the NuttX directory:
+NuttX provides ready-to-use board default configurations that enable the required config (from Kconfig) for a use scenario, such as Wi-Fi or I2C. To enter NuttX directory and list all available configurations for the ESP32 DevKitC V4 board you can use the following command:
```

Comment:
Please add a comma. It helps to visually separate the first part of the sentence saying what will happen from the second part of the sentence saying what you need to do to make it happen.

```suggestion
NuttX provides ready-to-use board default configurations that enable the required config (from Kconfig) for a use scenario, such as Wi-Fi or I2C. To enter NuttX directory and list all available configurations for the ESP32 DevKitC V4 board, you can use the following command:
```

If I may, I would like to propose further improvements to the sentence. The command `cd` is self-evident to most people, so the part `To enter NuttX directory` sounds like partonizing. How about this:

To list all available configurations for the ESP32 DevKitC V4 board, go to your `nuttx` directory and run:

Reply:
How about this:

"To list all available configurations for the ESP32 DevKitC V4 board, go to your `nuttxspace` directory and run:"

Directory structure looks like this:

```
.
└── nuttxspace/
    ├── nuttx
    └── apps
```

If we say go to your nuttx directory, we don't need to add `cd nuttx` command, I think. This is main purpose of this PR.